conf/config.yaml.sample

```diff
@@ -58,17 +58,46 @@ cortx:
     auth_admin: sgiamadmin
     auth_secret: s3_auth_admin_secret
   motr:
-    client_instances: 0
     interface_type: tcp
     interface_family: inet
     transport_type: libfab
     ios:
       endpoints:
-      - motr://data1-node1.cortx-cluster.com:3232
+      - inet:tcp:data-node1@28002
```

Comment:
Endpoints in the solution need to be in the URL format for consistency across all kinds of endpoints. 
Example: tcp://data1-node1.cortx-cluster.com:32320

More examples at
https://seagate-systems.atlassian.net/wiki/spaces/PUB/pages/611615649/CORTX+Configuration+Parameters

Reply:
Agree with this method, as it will uniform across all the endpoint formats and also utils library can be used to get the required info.

Reply:
@shriya-deshmukh : Can you change all the endpoints from inet:tcp:data-node1@28002  to tcp://data-node1@28002 ?

Reply:
Done